apps/dashboard/src/components/workflow-editor/nodes.tsx

```diff
@@ -364,13 +364,19 @@ export const AddNode = (_props: NodeProps<NodeType>) => {
                   navigate(
                     buildRoute(ROUTES.EDIT_STEP_TEMPLATE, {
                       stepSlug: data.steps[data.steps.length - 1].slug,
-                    })
+                    }),
+                    {
+                      state: { hideValidationErrorsOnFirstRender: true },
```

Comment:
ohhhhh i see, so we create issues on create but not render them. 
i thought the ticket was about not creating the issues on create.

Reply:
Correct! Issues must be there in the modelling but this is mostly a UX fix.